tools/wptrunner/wptrunner/browsers/chromium.py

```diff
@@ -0,0 +1,150 @@
+from . import chrome_spki_certs
```

Comment:
This file is almost entirely copied over from `chrome.py` and is very likely in need of some changes.

Reply:
Is this still the case?

Reply:
Yes it is. I'll go through today and try to discern what might need to be changed or different from the Chrome implementation of this file, but I imagine it's very similar.